filters/filters.txt

```diff
@@ -14,6 +14,7 @@
 ! which makes them candidates to be injected early in the page.
 ! Last imported: 2020-07-08
```

Comment:
This block of code is directly imported from EasyList with just `google.*` prepended. 

You should add new filters in new block at the bottom.

Reply:
Like [this](https://github.com/uBlockOrigin/uAssets/pull/8014/commits/c04a734f3fadb849a05906b479eba0e2eece0651)?